db/articles.js

```diff
@@ -0,0 +1,76 @@
+let articles = [];
+
+function all() {
```

Comment:
Would suggest to stick with the action naming convention of something like `getAllArticles`